src/Compilers/Core/Portable/MetadataReference/ModuleMetadata.cs

```diff
@@ -290,10 +290,7 @@ public ImmutableArray<string> GetModuleNames()
         /// </summary>
         /// <exception cref="ObjectDisposedException">Module has been disposed.</exception>
         /// <exception cref="BadImageFormatException">When an invalid module name is encountered.</exception>
-        internal MetadataReader MetadataReader
-        {
-            get { return Module.MetadataReader; }
-        }
+        public MetadataReader MetadataReader => Module.MetadataReader;
```

Comment:
Sure.  Though why would it throw?  Can you let me know so i can document this?  I'll also need to make my code resilient to that onthe other end.